addons/mail/static/src/models/composer/composer.js

```diff
@@ -852,6 +852,8 @@ function factory(dependencies) {
         }),
         attachments: many2many('mail.attachment', {
             inverse: 'composers',
+            // FIXME: this field should be readonly.
+            // The compute composer replace the value and trigger the error
```

Comment:
I still don't understand the issue here even with the message. Is it talking about `_computeComposers` in attachment? And if so, how is doing a replace not by-passing readonly? Computed are supposed to by-pass read-only, and so are inverse relations.